hist/hist/src/THStack.cxx

```diff
@@ -787,18 +787,19 @@ void THStack::Paint(Option_t *choptin)
          Int_t ny = nx;
          if (((nx*ny)-nx) >= npads) ny--;
          padsav->Divide(nx,ny);
+
```

Comment:
A possible improvement here is using the C++ range iterators. 
See method 1 in https://root.cern.ch/doc/master/classTList.html

Reply:
Thanks, but I think it is ok as it is now. It was coded like that since the beginning.